### PR TITLE
Fixed source directory bug when not set at the default (.)

### PR DESCRIPTION
**Pull request by pascalgulikers**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Follows the [authoring recommendations](https://github.com/tektoncd/catalog/blob/main/recommendations.md)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality of task changed or new task added)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [ ] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [ ] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [ ] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [ ] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [ ] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [ ] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

### CODE REVIEW

File: task/pytest/0.1/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/py
 * **PYTHON**: The used Python version, more precisely the tag for the Python image (_default_: `latest`)
 * **ARGS**: The additional arguments to be used with pytest
 * **SOURCE_PATH**: The path to the source code (_default_: `.`)
-* **REQUIREMENTS_FILE**: The name of the requirements file inside the source location (_default_: `requirements.txt`)
+* **REQUIREMENTS_FILE**: The name of the requirements file inside the source location (_default_: `requirements.txt`), with fallback to the requirements file in the root location
```
Comment (sbwsg) [inline task/pytest/0.1/README.md:24]: The _default_ should come at the end of the sentence rather than be included as a sub-clause of it.

**Review by vdemeester — COMMENTED** [review]
cc @chmouel

**Review by chmouel — APPROVED** [review]
thanks

**Comment by vinamra28** [comment]
hey can you please squash the commits? :slightly_smiling_face:

**Comment by pascalgulikers** [comment]
@vinamra28 Squashed :)

**Pull request merged**